### ISSUE TITLE
MaskablePPO does not remove invalid mask action

### ISSUE THREAD

**Issue by AbelSyx** [Stable-Baselines-Team/stable-baselines3-contrib#166]
### ❓ Question

Hey, thank you for your work on the MaskablePPO algorithm.
In the environment where I use Openai_ROS for initialized, I use PPO to remove invalid actions, but during the training process, I found that the model will still choose invalid action for output. My action space is a discrete action space with 90-dimensional data. My code is shown below.
``` python
from sb3_contrib.common.maskable.policies import MaskableActorCriticPolicy, MaskableMultiInputActorCriticPolicy
from sb3_contrib.common.wrappers import ActionMasker
from sb3_contrib.ppo_mask import MaskablePPO

def mask_fn(env) -> np.ndarray:
        # Do whatever you'd like in this function to return the action mask
        # for the current env. In this example, we assume the env has a
        # helpful method we can rely on.
        mask = env.valid_action_mask()
        valid_mask_action = np.array([bool(i) for i in mask])
        return valid_mask_action

# Init OpenAI_ROS ENV
env = StartOpenAI_ROS_Environment(task_and_robot_environment_name, robot_id=0, max_episode_steps=max_episode_steps, data_folder_path=data_folder_path)
env = Monitor(env, data_folder_path)

env = ActionMasker(env, mask_fn)  # Wrap to enable masking
model = MaskablePPO(MaskableMultiInputActorCriticPolicy, env, learning_rate=learning_rate, n_steps=n_steps, batch_size=batch_size, ent_coef=ent_coef, tensorboard_log=tensorboard_log_path, policy_kwargs=policy_kwargs, device="cuda", verbose=1)            

```
Please give me some inspiration or tips, thank you!

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/issues) in the repo
- [X] I have read the [documentation](https://sb3-contrib.readthedocs.io/en/master/)
- [X] If code there is, it is minimal and working
- [X] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by AbelSyx** [comment]
Thank you for your reply. I have already solved this problem. And I verified by retaining only one valid action.
